upb/hash/common.h

```diff
@@ -189,7 +189,7 @@ UPB_INLINE size_t upb_table_size(const upb_table* t) {
 
 UPB_INLINE bool upb_tabent_isempty(const upb_tabent* e) { return e->key == 0; }
 
-uint32_t _upb_Hash(const void* p, size_t n, uint64_t seed);
+UPB_API_INLINE uint32_t _upb_Hash(const void* p, size_t n, uint64_t seed);
```

Comment:
Why is this needed? This is an implementation detail inside upb.

Reply:
`Message_Hash` (from `message.c`) and `Msgval_GetHash` (from `convert.c`) call it directly, so FFI needs to be able to do the same.

Reply:
Reverted